src/app/core/market/api/cart/cart.model.ts

```diff
@@ -22,26 +22,34 @@ export class Cart {
       this.subTotal = new Amount(total);
     }
 
-    setShippingTotal(): void {
+    setShippingTotal(country: string): Amount {
       let total = 0.0;
       this.listings.forEach(listing => {
-        total += listing.internationalShippingPrice.getAmount();
+        total += country === listing.country ? (
```

Comment:
Wrong approach in this PR completely..

instead write a function the listing model:
listing.escrowPrice(country).getAmount()

Reply:
@kewde what's wrong with the approach ? Before escrowPrice is set using `setEscrowPrice` inside the model and that function is also used `setTotal` due to that reason I used the same functions using those variables!!